pom.xml

```diff
@@ -95,6 +95,27 @@
         </dependency>
 
 
+        <!-- https://mvnrepository.com/artifact/org.apache.kafka/kafka-clients -->
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
```

Comment:
You are already using `spring-kafka` remove it.

Reply:
Yes! I will remove this dependency.